_journal-configs/lancet.yaml

```
@@ -0,0 +1,26 @@
+name: The Lancet
+publisher: Elsevier
+url: https://www.thelancet.com/lancet/information-for-authors
+
+sizes:
+  single_column: { width_mm: 70,  max_height_mm: 230 }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)